### PR TITLE
Fix ner convert logits to probs

### PR DESCRIPTION
**Pull request by elboy3**
Fix failing examples e2e notebook for spacy inference. 

Link to failed notebook: https://github.com/rungalileo/examples/actions/runs/4503711642/jobs/7927264952

Successful run: https://console.enterprise-a.rungalileo.io/insights?dataframeColumns=color%3B%26confidence%3B%26data_error_potential%3B%26error_types%3B%26gold%3B%26pred%3B%26ranking%3B%26total_errors&groupedBy=pred&inferenceName=example1&metric=confidence&projectId=13ab66e6-dd33-4d00-b2d1-fb971f97ba2e&runId=5cc6e2dc-e029-4bfc-acee-501b7d53d24d&sortBy=confidence&sortDirection=asc&split=inference&taskType=2

Screen Shot 2023-03-23 at 4 59 11 PM

### CODE REVIEW

**Review by Ben-Epstein — APPROVED** [review]
nicee
